langchain/src/vectorstores/tests/elasticsearch.int.test.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-process-env */
```

Comment:
consider: also incorporating [unit tests](https://github.com/hwchase17/langchainjs/pull/1718/files#diff-e4c8fc644431432bfd8f99ebce64d13484f8434c0180769af36cc1d023654fd7R1) as well as [tests that involve metadata lookup](https://github.com/hwchase17/langchainjs/pull/1718/files#diff-4b771d4d6505aae476d14c91b8496fbf5ae591858597f482e30a77cc70a0bb8eR36)